src/server.py

```diff
@@ -337,8 +346,6 @@ def ConcludeGame(self, winner: str):
       self.PlayerB.win()
     return 0
 
-#if __name__ == "__main__":
-
```

Comment:
can we keep the two lines or delete the rest lines together? otherwise the rest lines would look like as if they were part of the class `Game`.

Reply:
I'm fairly certain that its just residual code from someones tests while developing that they have commented out. So I don't believe it will be in there for the final product, but I again I also didn't want to delete it on the off chance that someone needed it for reference or something.